upgrade/breaking-changes/9-0-0.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 9.0.0
+description: Handle the changes in the 9.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 9.0.0
+slug: changes-in-9-0-0
+position: 960
+---
+
+# Breaking Changes in 9.0.0
+
+Telerik UI for Blazor 9.0.0 targets .NET 8 and no longer supports .NET 6 and .NET 7. For more information, see [System Requirements](slug:system-requirements).
```

Comment:
It might be worth mentioning the deprecated templates as well.

Reply:
In the Breaking Changes articles, we normally list changes in the UI for Blazor product. The idea is to inform the customers what may break the components in their apps when they upgrade, so they can address the needed changes in advance.
The deprecation of the templates is a change in the VS and VS Code extensions that will not break the clients' apps. In fact, we deprecated the templates several releases ago but we have missed to document that for VS Code.